### ISSUE TITLE
Translatable (localized) int 24h handler (Abort, Retry, Ignore...)

### ISSUE THREAD

**Issue by mateuszviste** [SvarDOS/bugz#97]
The SvarCOM int 24h handler (Abort, Retry, Ignore...) has been implemented in #24, but it is hard-coded to EN-only.
This task would be to make it translatable.

It's relatively straight-forward: SvarCOM would need to know the offsets of the "Retry, Ignore..." strings within RMOD and update them accordingly, based on the LANG variable and the content of the SVARCOM.LNG file.

**Comment by mateuszviste** [comment]
for reference: how the int 24h handler looks like in the PL version of MS-DOS 7.x

![image](https://github.com/user-attachments/assets/69f2d3ef-a862-4956-822c-b8d0014ec3fd)

**Comment by mateuszviste** [comment]
[Implemented](http://svn.svardos.org/comp.php?repname=SvarDOS&compare[]=%2F@1880&compare[]=%2F@1881). Demo:

![image](https://github.com/user-attachments/assets/90f08371-dc87-4432-87c8-dc542cc4af3f)

Flagged the German translations as "unsure" since they are unlikely to be correct.

**Comment by boeckmann** [comment]
You have to work on the last one. Like it is now, it means something you want to do if no one is watching :D

<img width="482" alt="Bildschirmfoto 2024-08-15 um 15 10 13" src="https://github.com/user-attachments/assets/2c69667b-634f-453b-9545-abb8a86a9048">

**Comment by boeckmann** [comment]
I will take care of the german translation.

**Comment by mateuszviste** [comment]
> I will take care of the german translation.

Thanks :)

**Comment by mateuszviste** [comment]
Thanks for your lang fixes. I only modified this one, since the string was 1 byte too long. I hope it's still acceptable/understandable.

````
 # CRITICAL HANDLER (INT 24H) - EACH MESSAGE MUST BE NO LONGER THAN 15 BYTES!
-3.0:KRITISCHER FEHLR
+3.0:KRITISCH FEHLER
````

**Comment by boeckmann** [comment]
Oh, sorry for that. You may better replace it by `KRIT. FEHLER`.

**Comment by mateuszviste** [comment]
> You may better replace it by `KRIT. FEHLER`.

Fixed. And then I released SvarCOM 2024.4 so SvarDOS is "EDR-ready" now. :)
